### ISSUE TITLE
Don't prompt rebuild when changing PAPI metrics

### ISSUE THREAD

**Issue by jlinford** [ParaToolsInc/taucmdr#124]
[TAU] Application rebuild required:
[TAU]   - metrics changed from [TIME,PAPI_NATIVE:LLC_MISSES] to [TIME,PAPI_NATIVE:MEM_UOPS_RETIRED:ALL_STORES]

**Comment by jlinford** [comment]
Working as of a8ed345e19f4044711f2f542bb0af89dcd6b46f8

**Comment by jlinford** [comment]
This issue is back as of 5f623965c7f2c976ec6b02742ce88862ee18d9cb.  

```
tau meas cre load_store_ins --metrics="TIME,PAPI_LD_INS,PAPI_SR_INS,PAPI_TOT_INS" --compiler-inst never --source-inst automatic
tau sel load_store_ins
tau meas cre floating_pt_ins --metrics="TIME,PAPI_FP_INS,PAPI_TOT_INS" --compiler-inst never --source-inst automatic
tau sel floating_pt_ins
[TAU] Created a new experiment 'east03-mm-floating_pt_ins'
[TAU] Selected experiment 'east03-mm-floating_pt_ins'.
[TAU] Application rebuild required:
[TAU]   - metrics changed from [TIME,PAPI_LD_INS,PAPI_SR_INS,PAPI_TOT_INS] to [TIME,PAPI_FP_INS,PAPI_TOT_INS]
```

**Comment by khsa1** [comment]
Fixed by f4f50e49d912f222675bd59810764ecfaab7605c.
